com.unity.ml-agents/Runtime/AssemblyInfo.cs

```diff
@@ -2,3 +2,4 @@
 
 [assembly: InternalsVisibleTo("Unity.ML-Agents.Editor.Tests")]
 [assembly: InternalsVisibleTo("Unity.ML-Agents.Editor")]
+[assembly: InternalsVisibleTo("Unity.ML-Agents.Extensions")]
```

Comment:
Possibly controversial; I added this to get timers. But it's something we could use in the future, e.g. public actuators in extensions, but keep IActuator internal.